### PR TITLE
Merge version 1.0.0

### PR DESCRIPTION
**Pull request by tekkabroley**
Merging data-sentinel 1.0.0 into into Master branch.

Added sql directory to contain core sql query building logic. 
sql/core.py - contains logic to build select and aggregate queries.
sql/helpers.py - a collection of query and parameter formatting functions
sql/measures.py - contains logic to build sql queries for measuring data integrity
sql/tests.py - contains logic to build data quality tests

Added multi_proportion_duplicate_sql_test to datasentinel/tests.py. This function tests a collection of columns simultaneously for data duplication. 

Refactored datasentinel/log.py to expose three log functions which can be used in development. All three methods return a logging.RootLogger object.
1. get_file_logger - writes log messages to file_path
2. get_stream_logger - writes to stream. If None is specified then writes to sys.stderr
3. get_null_logger - swallows messages. This is the default logger if None is specified.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,178 @@
...
+
+## Prerequisites
+Python 3.6+
+
+sqlalchemy>=1.3.19
+
+psycopg2
+
+## Installing
+`pip install data-sentinel`
```
[the remark this answers is not in the document]
Comment (TomQbiz) [reply README.md:19]: I think we need to run:
`pip install .`

**Review by TomQbiz — CHANGES_REQUESTED** [review]
How do I run the example?
Do we need to setup a venv?

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (tekkabroley) [reply README.md:19]: Right now the data-sentinel package only exists in test.pypi.org. It won't exist in pypi.org until after the merge with master. 

To install the test version use the
`pip install -i https://test.pypi.org/simple/ data-sentinel`

**Comment by tekkabroley** [comment]
@TomQbiz  to run the test scripts located in the tests directory you need to create and load the test_schema.Orders, test_schema.Contacts and test_schema.Earthquakes tables which are defined by the sql scripts in the test_data directory.

**Pull request merged**